src/app/app.component.ts

```diff
@@ -270,6 +270,9 @@ export class AppComponent {
   }
 
   ngOnInit() {
+    // Sitewide survey
+    this._script.loadScript('ethnio-survey')
```

Comment:
Adding this script to the app.component appears to be the way to make this sitewide